src/hotspot/share/gc/z/zRelocate.cpp

```diff
@@ -1167,26 +1164,30 @@ class ZRelocateTask : public ZRestartableTask {
 
     const auto do_forwarding_one_from_iter = [&]() {
       ZForwarding* forwarding;
-      const uint32_t start_node = ZNUMA::id();
-      uint32_t current_node = start_node;
 
-      for (uint32_t i = 0; i < num_nodes; i++) {
+      while (true) {
```

Comment:
ZGC exclusively uses `for (;;)` for infinite loops (with the exception of one `do { [...] } while(true);`)
```suggestion
      for (;;) {
```